public/index.html

```diff
@@ -24,26 +21,22 @@
       <div class="u-padding__x--0@lg u-padding__x--0@lg">
         <div class="o-grid o-grid--nowrap@lg">
           <div class="o-grid-12 o-grid-auto@lg o-order-1 o-order-2@lg">
-            <article
-              id="article"
-              class="c-article c-article--readable-width s-article u-clearfix"
-            >
-              <h1 class="c-typography c-typography__variant--h1">
-                Mitt Helsingborg Bokningstjänst
-              </h1>
+            <article id="article" class="c-article c-article--readable-width s-article u-clearfix">
+              <h1 class="c-typography c-typography__variant--h1">Mitt Helsingborg Bokningstjänst</h1>
               <p>En testsida för Mitt Helsingborgs bokningstjänst.</p>
               <br />
             </article>
             <div class="sidebar-content-area o-grid">
               <div class="modularity-mod-mhbooking">
                 <div class="c-card c-card--default c-card--ratio-4-3">
                   <div class="c-card__body">
-                    <h2
-                      class="c-typography c-card__heading c-typography__variant--p"
-                    >
-                      Boka ditt möte
-                    </h2>
-                    <div data-endpoint="xxxxx" id="mh-mount-booking"></div>
+                    <h2 class="c-typography c-card__heading c-typography__variant--p">Boka ditt möte</h2>
+                    <div
+                      data-endpoint="xxxxx"
+                      data-sharedmailbox="datatorget_testgrupp@helsingborgdemo.onmicrosoft.com"
```

Comment:
Not sure if data-sharedmailbox and data-subject should be hard-coded here?

Reply:
Do you meen that should hide it as we do with `data-endpoint` and configure it locally? This file is not a part of the application and the `div` is just a mountingpoint for the app.

Reply:
Yes, thats exactly what I meant.

Reply:
Okey fixed :)